lisa/util/__init__.py

```diff
@@ -418,6 +418,17 @@ def enable(self) -> None:
         self._switch(True)
 
 
+class LisaVersionInfo(VersionInfo):
```

Comment:
Please either split the version change into a separate commit or PR. If you want to split by commits, make all commits meaningful in the PR.

Reply:
Updated the commits